src/unittest/python/test_api.py

```diff
@@ -20,45 +21,42 @@ def server_is_up():
         else:
             # Quasi check for server status
             return all([bool(txt) for txt in ['api', 'is', 'online'] if txt in str(status_request)])
-        
+
     def test_credential_store(self):
         from os import path
         if self.server_is_up():
-            with Api('/account', auth=True) as api:
+            with Api('/account') as apiInstance:
```

Comment:
Minor nit: My PEP8 linter yells at me about `apiInstance` not being snake case (i.e. `api_instance`).